imports/plugins/included/ui-search/lib/components/searchModal.js

```diff
@@ -20,6 +24,134 @@ class SearchModal extends Component {
     value: PropTypes.string
   }
 
+  constructor(props) {
```

Comment:
As no initial state value uses `props`, the constructor is not necessary.
You can initialize the state as a variable in the class without using the constructor. Therefore, delete the constructor and  initialize state as:
```
state = {
  allProducts: [],
  ...
  highPriceFilter: 0
};